src/reductive.re

```diff
@@ -45,17 +45,17 @@ module Provider = {
     unsubscribe: option (unit => unit)
   };
   let createMake ::name="Provider" (store: Store.t 'action 'state) => {
-    let innerComponent = ReasonReact.statefulComponent name;
```

Comment:
Is this necessary?

Reply:
It gives a warning with reason-react@0.2.4 asking you to use reducerComponent instead.